DI-1299/compare_b37_and_b38.py

```
@@ -0,0 +1,477 @@
+    Returns
+    -------
+    df_melted : pd.DataFrame
+        melted df with multiple rows per sample - all variables are in
+        'variable' column and all values are in 'value' column
+    """
+    col_stubnames = list(
+        set(
+            col.removesuffix("_GRCh37").removesuffix("_GRCh38").strip()
+            for col in mismatches.columns
+            if col != sample_column
+        )
```

Ruff (0.8.2)

Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)